server/src/helpers/generateToken.ts

```diff
@@ -0,0 +1,19 @@
+import jwt from 'jsonwebtoken';
+
+const generateToken = (payload) => {
+  const tokenGenerate = new Promise((resolve, reject) => {
+    jwt.sign(
+      { role: 'user', payload },
```

Comment:
```suggestion
      payload,
```
SO we can use `generateToken` to both admin and user auth